COMMUNITY.md

```diff
@@ -0,0 +1,62 @@
+# eksctl community
+
+Welcome to the eksctl community. We are very happy to have you here :tada: .
+
+This document contains the following information:
+
+- [Who the core team members are](#core-team)
+- [How to join the community](#join-us) and get in touch with the core team
+- [How eksctl is run](#how-eksctl-is-run) and how decisions are made
+- [Community meetings](#community-meetings)
+- [Community rules](#community-rules)
+
+Ready to start contributing? Head over to our [contribution docs](CONTRIBUTING.md) :man_technologist:.
+
+## Core team
+
+The core team of maintainers are Mike Beaumont ([@michaelbeaumont](https://github.com/michaelbeaumont)),
+Chetan Patwal ([@cPu1](https://github.com/cPu1)),
+Jake Klein ([@aclevername](https://github.com/aclevername))
+and Claudia Beresford ([@Callisto13](https://github.com/Callisto13)).
+
+## Join us!
+
+The project uses Slack. If you get stuck or just have a question then you are encouraged to join the
+[Weave Community](https://slack.weave.works/) Slack workspace and use the
+[#eksctl](https://weave-community.slack.com/messages/eksctl/) channel and/or the [mailing
+list][maillist].
+
+We use the mailing list for some discussion, for sharing documents and for calendar invites.
+
+[maillist]: https://groups.google.com/forum/#!forum/eksctl
+
+## How eksctl is run
+
+Eksctl is an open-source project built to interact with a closed-source product.
+This means that 99% of the project operates like any other open-source one,
+with 1% of things happening in slightly secretive fashion as the maintainers
+work with the folks at Amazon to synchronise the release of cool new EKS stuff :sunglasses: .
+
+For that 99%, we strive to keep development open, transparent, and unbiased.
+Work is driven by community needs and user feedback, and we welcome all contributions
```

Comment:
This isn't sitting quite right with me. how about something like:

"Work is driven through partnership with AWS and the community needs. We welcome all contributions and user feedback to the project."